BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs

```diff
@@ -66,7 +65,15 @@ async void Start()
             }
         }
 
-        public void AnchorsChanged(ARAnchorsChangedEventArgs eventArgs)
+        protected void OnDisable()
+        {
+            if (m_arAnchorManager != null)
+            {
+                m_arAnchorManager.anchorsChanged -= AnchorsChanged;
```

Comment:
Using events to keep two lists up-to-date can be really hard, with this disconnection to the manager, are we sure this scene will be able to keep latest view of all anchors?   Or are we assuming the "enable/disable" of this component will go together with the unity scene which will destroy all gameobjects related to ARAnchor?

Reply:
I think my assumption was that it'll typically go together with the whole Unity scene, but you make a good point about keeping the latest view. I believe I can query all existing trackables from the manager, so maybe it's worth doing that in `OnEnable` to get back up-to-date?

Reply:
Added a new codepath in `OnEnable` that reads all currently tracked anchors from the manager and starts tracking them if needed.